### PR TITLE
Add join table for visits

### PR DESCRIPTION
**Pull request by iankiema**
The following files have been updated 
-Data.sql
-Queries.sql 
-Schema.sql

Below are the query snapshots
details for the most recent visit;animal information,vet informaition and date of visit
speciality Maisy should consider getting
Visits that were with a vet that did not specialize in that animals species
Maisy smith first visit
animal that has most visits to vets
All animals that visited Stephanie Mendez between April 1st and August 30th, 2020
All vets and their specialities including vets with no specialities
Different animals seen by Stephanie Mendez
last animal seen by william tatcher

### CODE REVIEW

File: schema.sql [hunk 1]
```diff
@@ -35,3 +35,26 @@ drop column species;
 alter table animals
 add column species_id integer references species(species_id),
 add column owner_id integer references owners(owner_id);
+
+
+/* Create table named 'Vets' */
+create table vets(
+    vet_id serial primary key,
+    vet_name varchar,
```
Comment (Shedrack-Sunday) [inline schema.sql]: - [ ] Setting limits on VARCHAR columns in a database ensures data integrity, improves performance, and enhances security by preventing overly long or inappropriate data. Kindly add a limit to the name field 👍

File: queries.sql [hunk 2]
```diff
@@ -128,4 +128,92 @@ from owners o
...
+    select an.species_id, count(*) as visit_count
+    from visits v
+    inner join animals an on v.animal_id = an.animal_id
+	inner join vets vt on v.vet_id = vt.vet_id
+	where vt.vet_name = 'Maisy Smith'
+	group by an.species_id
+	order by visit_count desc
+	limit 1
+) most_visited_species
+join species s on most_visited_species.species_id = s.species_id;
```
Comment (Shedrack-Sunday) [inline queries.sql]: - [ ] This query was duplicated twice. you're meant to search `Details for most recent visit: animal information, vet information, and date of visit`, however, you did a query similar to the last query. Kindly look at the sample below

### Sample:
```
  SELECT animals.*, vets.*, vistis.visit_date
  FROM visits
  LEFT JOIN animals ON animals.id = visits.animal_id
  LEFT JOIN vets ON vets.id = visits.vet_id
  ORDER BY visits.visit_date DESC
  LIMIT 1;
  ```
- Note: The above is just a sample. your query should make sure to give the right results

**Review by Shedrack-Sunday — CHANGES_REQUESTED** [review]
Hello @iankiema           👋 

# CHANGES REQUIRED ♻️

Impressive work on the project so far👏
You've done extremely well so far in this project,  However, there are some adjustments you need to make to move to the next project. 💯



## Highlights   
- Good use of GitHub features ✔️   
- Nice work with the SQL  features ✔️ 
- Nice use of query parameters✔️ 

# Required Changes  🌵  

- [x] You've done well, However, kindly follow the suggestion on the right query pattern properly so your impressive work will have a better implementation 👍 


_Check the comments for a detailed review 

# Optional suggestions 

Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing really improves your application.

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise.

As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s).

File: queries.sql (hunk 2 above)
Comment (iankiema) [reply queries.sql]: Most recent visit ;animal information, vet information and visit date
most recent visit animal info vet info date of visit

**Review by KDlamini — APPROVED** [review]
### Hi @iankiema 🙋‍♀️,  

Excellent work! 💯 
You have done a great job implementing all the required changes for adding join tables for the Vet clinic database. 
Your feature is complete! There is nothing else to say other than... it's time to merge it. :shipit: 
Congratulations!!!🎊 🥳 

## Highlights 🌞 

✔️  Professional pull request
✔️  Correct use of Gitflow 
✔️  All required SQL commands are in their respective files
✔️  Join tables and queries are well implemented

# Status: Approved ✔️ 

Great-job-smiley

Cheers 🥂  and Happy coding!!! 👯 
Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Remember to tag me @KDlamini  in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think the code review was unfair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**